src/subscription_manager/cache.py

```diff
@@ -231,7 +231,7 @@ def load_status(self, uep, uuid, on_date=None):
             self.last_error = ex
             log.error("Bad identity, unable to connect to server")
             return None
-        # all of the abover are subclasses of ConnectionException that
+        # all of the above are subclasses of ConnectionException that
```

Comment:
while it's a good typo fix, i'd do it in a separate PR (usually better avoid unrelated changes)

Reply:
put back and made note to open a PR, and going to include the typo in migrate.py also